instrumentation/netty-4.0/src/test/java/com/splunk/opentelemetry/netty/v4_0/NettyInstrumentationTest.java

```diff
@@ -120,15 +124,9 @@ private static void assertHeaders(Response response, String serverTimingHeader)
 
   private static void assertServerTimingHeaderContainsTraceId(String serverTimingHeader)
       throws InterruptedException, TimeoutException {
-    exporter.waitForTraces(1);
-
-    var traces = exporter.getTraces();
-    assertEquals(1, traces.size());
-
-    var spans = traces.get(0);
-    assertEquals(1, spans.size());
+    instrumentation.waitForTraces(1);
 
-    var serverSpan = spans.get(0);
+    var serverSpan = instrumentation.spans().get(0);
```

Comment:
Will this fail if 1000 traces come back?  Seems like we were checking explicitly before but are not now...?

Reply:
Hmm, you're right. I'll revert this code to assert trace/span counts